### DISCUSSION TITLE
This user account is blocked

### DISCUSSION THREAD

**Discussion by erikahumadakandc** [vercel/community#3829] [Account Issue] [answered]
### Question

I received an invitation to collaborate on a project so I created the account using the GitHub account for that project. After that, I just received this error message. Could you please give me a hand? This is a priority.

Thanks!

**Comment by GeovanniFSantos** [comment]
Se você conseguir resolver, passa para a gente aqui no FORUM. Porque eles só fala que não pode discutir isso aqui e manda um link...

**Comment by amyegan** [comment, marked as answer]
Hey @erikahumadakandc. Sorry you're having trouble with account creation. We don't discuss account information in public because of safety and security concerns. If you email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you are looking to deploy with Vercel, that will create a new support ticket.

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Reply by erikahumadakandc** [reply]
I sent an email a few days ago and we haven't received any response, we have a pro plan, We need your help ASAP

**Comment by itzmiftah** [comment]
i get a message this user account is blocked . please resolve it.

**Reply by Developer0000000** [reply]
i get also

**Comment by Tuziking** [comment]
I also get a message this user account is blocked and I sent an email a few days ago, but they don't give me a response

**Comment by night-slayer18** [comment]
please please fix this issue or else vercel will die soon enough why the heck is it having too much protection layer issue

**Comment by jtvberg** [comment]
4 follow ups, over 3 weeks and no response in my case.
Netlify it is.
